src/generic-container/generic-container.test.ts

```diff
@@ -509,6 +510,39 @@ describe("GenericContainer", () => {
     await container.stop();
   });
 
+  it("should create container with volume", async () => {
+    const volume: Volume = { name: "/volumes/new-volume" };
+
+    const container = await new GenericContainer("cristianrgreco/testcontainer:1.1.12")
+      .withExposedPorts(8080)
+      .withVolumes(volume)
+      .start();
+
+    const dockerContainer = await getContainerById(container.getId());
+    const containerInfo = await dockerContainer.inspect();
+    await container.stop();
+    expect(containerInfo.Mounts.length).toBe(1);
+    expect(containerInfo.Mounts[0].Destination).toBe(volume.name);
+  });
+
+  it("should create container with array of volumes", async () => {
```

Comment:
There isn't any difference in behaviour if you pass one or multiple, so this test can be removed